controllers/statusreport/statusreport_adapter.go

```diff
@@ -65,7 +62,7 @@ func NewAdapter(snapshot *applicationapiv1alpha1.Snapshot, application *applicat
 // EnsureSnapshotTestStatusReported will ensure that integration test status including env provision and snapshotEnvironmentBinding error is reported to the git provider
 // which (indirectly) triggered its execution.
 func (a *Adapter) EnsureSnapshotTestStatusReported() (controller.OperationResult, error) {
```

Comment:
There's a similar sounding function within the integrationpipeline_controller that I'm getting confused with, should we update their names to avoid confusion? something like this?
```suggestion
func (a *Adapter) EnsureSnapshotTestStatusReportedToGitHub() (controller.OperationResult, error) {
```

Reply:
Yes, it makes sense to me. Thanks.

Reply:
Thanks!

Reply:
Updated.